nextcord/client.py

```diff
@@ -2568,8 +2581,10 @@ def message_command(
         name_localizations: Dict[Union[:class:`Locale`, :class:`str`], :class:`str`]
             Name(s) of the command for users of specific locales. The locale code should be the key, with the localized
             name as the value
-        guild_ids: Iterable[:class:`int`]
-            IDs of :class:`Guild`'s to add this command to. If unset, this will be a global command.
+        guild_ids: Optional[Iterable[:class:`int`]]
+            IDs of :class:`Guild`'s to add this command to. If set to :attr:`utils.MISSING` and `default_guild_ids` is
+            set, then those default guild ids will be used instead. If both of those are unset, then the command will
+            be a global command. Defaults to `MISSING`.
```

Comment:
```suggestion
            IDs of :class:`Guild`'s to add this command to. If set to :attr:`utils.MISSING` and ``default_guild_ids`` is
            set, then those default guild ids will be used instead. If both of those are unset, then the command will
            be a global command. Defaults to ``MISSING``.
```
and similar comment about `` :attr:`utils.MISSING` ``